libs/NfcCoreLib/inc/phNfcTypes.h

```diff
@@ -250,9 +252,12 @@ typedef struct phNfc_sDeviceCapabilities
 
     struct
     {
-        BitField_t          AidBasedRouting : 1;       /**<Supported if the bit is set to 1b*/
-        BitField_t          ProtocolBasedRouting : 1;  /**<Supported if the bit is set to 1b*/
-        BitField_t          TechnBasedRouting : 1;     /**<Supported if the bit is set to 1b*/
+        BitField_t          ForceBasedRouting : 1;     /**<Supported if the bit is set to 1b*/
```

Comment:
ForceRouting functionality is not implemented in NfcCx as I understand? If yes, please leave a comment next to the field declaration specifying that

Reply:
Yes this is correct ForceRouting is not implemented in the NfcCx.
I have some changes in staging for this but I realized it wasn't making sense until we can signal to the NfcCx a device power mode change with the same granularity. I am holding on that topic and will had a comment to the field declaration.